pmoves/design/build/tokens.darkxside-skin.css

```
@@ -0,0 +1,27 @@
+/* generated by pmoves/design/generate.py — do not edit */
+:root[data-theme="darkxside-skin"] {
+  --pm-accent: #E11D48;
+  --pm-accent-2: #7C3AED;
+  --pm-accent-soft: #FB7185;
+  --pm-bg: #0a0608;
+  --pm-bg-tint: #140a0e;
+  --pm-border-strong: rgba(255,255,255,0.15);
+  --pm-border-subtle: rgba(255,255,255,0.08);
+  --pm-font-body: 'Exo 2', system-ui, sans-serif;
+  --pm-font-display: Orbitron, 'Exo 2', system-ui, sans-serif;
```

Stylelint (17.13.0)

[error] Expected "Orbitron" to be "orbitron" (value-keyword-case)

(value-keyword-case)